test/utils.test.ts

```diff
@@ -650,4 +583,38 @@ describe('utils', function () {
       expect(result).to.be.true;
     });
   });
+
+  describe('getRelayClientGenerator', function () {
```

Comment:
We should include a test where it fails, for example that the list is undefined. Or maybe we should handle that error in the generator.